ruma-client-api/src/r0/backup/add_backup_key_session.rs

```diff
@@ -0,0 +1,67 @@
+//! [PUT /_matrix/client/r0/room_keys/keys/{roomId}/{sessionId}](https://matrix.org/docs/spec/client_server/unstable#put-matrix-client-r0-room-keys-keys-roomid-sessionid)
+
+use js_int::UInt;
+use ruma_api::ruma_api;
+use ruma_identifiers::RoomId;
+
+use super::KeyData;
+
+ruma_api! {
+    metadata: {
+        description: "Store several keys in the backup.",
+        method: PUT,
+        name: "add_backup_key_session",
+        path: "/_matrix/client/r0/room_keys/keys/:room_id/:session_id",
+        rate_limited: true,
+        authentication: AccessToken,
+    }
+
+    #[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
```

Comment:
The request and response blocks no longer need an attribute for non-exhaustiveness.